ores/score_processors/celery.py

```diff
@@ -174,10 +198,42 @@ def from_config(cls, config, name, section_key="score_processors"):
             metrics_collector = None
 
         timeout = section.get('timeout')
+        queue_maxsize = section.get('queue_maxsize')
         application = celery.Celery('ores.score_processors.celery')
         application.conf.update(**{k: v for k, v in section.items()
-                                   if k not in ('class', 'timeout')})
+                                   if k not in ('class', 'timeout',
+                                                'queue_maxsize')})
 
         return cls(scoring_contexts, application=application, timeout=timeout,
-                   score_cache=score_cache,
+                   score_cache=score_cache, queue_maxsize=queue_maxsize,
                    metrics_collector=metrics_collector)
+
+
+REDIS_URL_RE = re.compile(
```

Comment:
Seems like it might be more complicated to handle the uri parts.  

```
>>> parse.urlparse("redis://foobar@localhost/0")
ParseResult(scheme='redis', netloc='foobar@localhost', path='/0', params='', query='', fragment='')
```